File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCount.java
package app.documents.core.network.manager.models.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import app.editors.manager.mvp.models.base.Base;

public class ResponseCount extends Base {

    @SerializedName(KEY_RESPONSE)
    @Expose
    private int response;

    public int getResponse() {
        return response;
    }

    public void setResponse(int response) {
        this.response = response;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/response/ResponseCount.kt
package app.documents.core.network.manager.models.response

import app.documents.core.network.common.models.BaseResponse
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ResponseCount(
    @SerializedName(KEY_RESPONSE)
    @Expose
    var response: Int = 0
) : BaseResponse()